### DISCUSSION TITLE
Higher Geography Committee Reboot

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4810] [Announcements]
Calling all Arctos users that are interested in Geography and Locality data! A recent slew of issues and data quality queries has led me to think that we need to start monthly meeting to work on Geography/Locality issues in Arctos. I have rounded up all of the issues that I think are pertinent and placed them in the [Geography in Arctos](https://github.com/ArctosDB/arctos/projects/2) project and I plan to start the discussion during Arctos Office Hours on July 26 at 1 PM MDT in the Arctos Zoom Room.

**Comment by mkoo** [comment]
Good idea to revitalize Geo group and use this time slot for some focused work! I would attend with no prodding but I will be at the Ichs and Herps conference that week so could this be postponed and find another time?!

**Reply by jtgiermakowski** [reply]
I'm very interested in this group but also will be at ASIH that week, from Wed onwards. Maybe we can meet Tuesday and I'll look for @mkoo  in Spokane?

**Reply by mkoo** [reply]
Tom, we should definitely meet up in Spokane!
I'll work on a regular meeting time for the geo-group too. Look out for a poll sent to those interested in the team in email directly.

**Comment by Jegelewicz** [comment]
@mkoo perhaps you would like to just take on the Committee and set a monthly meeting day/time? I have too many committees to wrangle as it is, but this need some work - it has been set aside for too long.

**Comment by sharpphyl** [comment]
Suggestions for starter topics that would help our collection.

reopen [#3043](https://github.com/ArctosDB/arctos/issues/3043).  The West Indies is still in our higher geography.
reopen [#2876](https://github.com/ArctosDB/arctos/issues/2876).  Add Waterbody as a separate field.
review [#128](https://github.com/tdwg/dwc-qa/issues/128) for a related supporting comment. 
tackle [#2374](https://github.com/ArctosDB/arctos/issues/2374) on marine/offshore locations
reopen [#1107](https://github.com/ArctosDB/arctos/issues/1107) which discusses using EEZs
one more - reopen [#1291](iDigBio Flags on Continent #1291) unless you think it was actually resolved.

**Comment by mkoo** [comment]
Email with when2meet sent to AWG. @sharpphyl @jtgiermakowski @dustymc (when you're back) @Jegelewicz please fill out your availability-- we can arrange a regular monthly meeting next. Open to all interested! Hope to see some of the new Arctos users too!

**Comment by dustymc** [comment]
I'm not sure I'm ever going to find the bottom of my inbox at this point, but YES YES YES to the committee; just add it to the Arctos calendar and I'll find a way to make the time.

I now have tools capable of addressing real problems instead of flopping around from symptom to symptom, but I can't fully deploy them without some support from the Community. I think this is the single most important/impactful thing on my plate at the moment.

**Comment by ewommack** [comment]
I'm out of the field for a bit, and would be happy to join. I'm still not sure I have a lot of experience in the area, but I know it can be really important.

**Comment by mkoo** [comment]
Quick notice for all: on the Arctos calendar now, a reboot Geography WG
meeting on 8/11 at 12 pm PT.
I know Tom is out in the field but we can either use this time or choose
another as a monthly regular meetup to hash out geography issues.
Agenda to come closer to meeting time!
MK

**Comment by genevieve-anderegg** [comment]
@Jegelewicz Just getting back to the office after the holidays - I know we voted to switch this month's Arctos Geography Committee meeting from Jan 1st to Jan 8th (today) but it isn't on the calendar and I don't think an announcement was sent out - want to still go forward with it or should we wait to February?

**Comment by Jegelewicz** [comment]
@genevieve-anderegg probably wait until February. Do you know how to/are you able to add things to the calendar?

**Comment by genevieve-anderegg** [comment]
Sounds good to me - I don't think we'll have a lot to talk about today so that seems like the right idea. I don't think I have edit access to the calendar.

**Comment by Jegelewicz** [comment]
When are the meetings supposed to be scheduled for?

**Comment by genevieve-anderegg** [comment]
12 PM PT (1PM Mountain) first monday of every month!

**Reply by Jegelewicz** [reply]
OK - the February meeting is already there.

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain! @sharpphyl @Jegelewicz @mkoo @falco-rk @happiah-madson @dustymc

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain! @sharpphyl @Jegelewicz @mkoo @falco-rk @happiah-madson @dustymc

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain! @sharpphyl @Jegelewicz @mkoo @falco-rk @happiah-madson @dustymc

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain! @sharpphyl @Jegelewicz @mkoo @falco-rk @happiah-madson @dustymc

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain! @sharpphyl @Jegelewicz @mkoo @falco-rk @happiah-madson @dustymc

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain! @sharpphyl @Jegelewicz @mkoo @falco-rk @happiah-madson @dustymc

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain! @sharpphyl @mkoo @falco-rk @happiah-madson @dustymc @ArctosDB/geo-group

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain! @sharpphyl @mkoo @falco-rk @happiah-madson @dustymc @ArctosDB/geo-group Hope to see you all there!

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain! @sharpphyl @mkoo @falco-rk @happiah-madson @dustymc @ArctosDB/geo-group Hope to see you all there! 
Main topic will be our new Geography Proposal!!! https://github.com/ArctosDB/internal/issues/366
Agenda Link: https://docs.google.com/document/d/1tVmulmkFzN7P_IliFkeiOvMfKTAB_CCzG55ZxX0NFMI/edit?usp=sharing

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain! @sharpphyl @mkoo @falco-rk @happiah-madson @dustymc @ArctosDB/geo-group Hope to see you all there!

Agenda Link: https://docs.google.com/document/d/1tVmulmkFzN7P_IliFkeiOvMfKTAB_CCzG55ZxX0NFMI/edit?usp=sharing

**Reply by Jegelewicz** [reply]
So many Arctos repos are now invisible to me that I cannot understand what is being discussed in the agenda. It might be just me, but I suspect that a lot of other community members lack access and therefore could not be aware of developments in Arctos.

**Reply by mkoo** [reply]
@Jegelewicz If you want to request access, just let the officers know but all AWG members have access so shouldnt be a problem as long as they are logged into Github. All requests start off in the public forum so that is viewable by the world!

**Reply by Jegelewicz** [reply]
I am unable to access any of the provided links in the agenda

https://github.com/ArctosDB/internal/issues/366
https://github.com/ArctosDB/arctos-assets/issues/11
https://github.com/ArctosDB/internal/issues/335

> All requests start off in the public forum so that is viewable by the world!

If they are subsequently moved to a private repo, there are clearly community members (me) who cannot view them.

> all AWG members have access

OK - I am doing work for an Arctos Community Member - may I also have access? What makes one part of the AWG?

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain! @mkoo @falco-rk @happiah-madson @dustymc @ArctosDB/geo-group Hope to see you all there!

Agenda Link: https://docs.google.com/document/d/1tVmulmkFzN7P_IliFkeiOvMfKTAB_CCzG55ZxX0NFMI/edit?usp=sharing

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain/ 3PM Eastern! @mkoo @falco-rk @happiah-madson @dustymc @ArctosDB/geo-group Hope to see you all there!

Agenda Link: https://docs.google.com/document/d/1tVmulmkFzN7P_IliFkeiOvMfKTAB_CCzG55ZxX0NFMI/edit?usp=sharing

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain/ 3PM Eastern! @mkoo @falco-rk @happiah-madson @dustymc @ArctosDB/geo-group Hope to see you all there!

Agenda Link: https://docs.google.com/document/d/1tVmulmkFzN7P_IliFkeiOvMfKTAB_CCzG55ZxX0NFMI/edit?usp=sharing

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain/ 3PM Eastern! @mkoo @falco-rk @happiah-madson @dustymc @ArctosDB/geo-group Hope to see you all there!

Agenda Link: https://docs.google.com/document/d/1tVmulmkFzN7P_IliFkeiOvMfKTAB_CCzG55ZxX0NFMI/edit?usp=sharing

**Reply by mkoo** [reply]
Sorry I cant make it today!

**Comment by genevieve-anderegg** [comment]
Arctos Geography Committee Meeting today at 12PM Pacific/ 1PM Mountain/ 3PM Eastern! @mkoo @falco-rk @happiah-madson @dustymc @ArctosDB/geo-group Hope to see you all there!

Agenda Link: https://docs.google.com/document/d/1tVmulmkFzN7P_IliFkeiOvMfKTAB_CCzG55ZxX0NFMI/edit?usp=sharing
